.babelrc

```diff
@@ -1,9 +1,14 @@
 {
   "plugins": [
-    "transform-object-rest-spread",
-    "transform-object-assign",
-    "transform-class-properties",
+    ["@babel/plugin-transform-flow-strip-types", {"requireDirective": true}],
+    "add-module-exports",
```

Comment:
I think this is used so that we won't have to use `.default` everywhere.

Reply:
Is this something we wouldn't need if we cleaned up our files to be more idiomatic to ES6 modules / exports?

Reply:
Yea. Starting from Babel 6, we'll have to explicitly specify `.default`. 

But I think this plugin might be avoided with a few manual changes. I'll look into it.

Reply:
isn't this only applicable when a module is `require`d? The codebase seems to be using imports. Could you provide an example of what fails without this plugin?

Reply:
> isn't this only applicable when a module is `require`d? The codebase seems to be using imports. Could you provide an example of what fails without this plugin?

yea you're right, I realized this too late. It's only required by the files in examples (like [ColumnGroupsExample](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/examples/ColumnGroupsExample.js)) which is probably why the site build failed.

I'm opening up a new PR, where this plugin won't probably be required. Looking into it.